aicsimageio/tests/test_transforms.py

```diff
@@ -30,6 +30,64 @@
         (da.zeros((6, 200, 400)), "ZYX", "STCZYX", {}, (1, 1, 1, 6, 200, 400)),
         (np.zeros((6, 200, 400)), "ZYX", "ZCYSXT", {}, (6, 1, 200, 1, 400, 1)),
         (da.zeros((6, 200, 400)), "ZYX", "ZCYSXT", {}, (6, 1, 200, 1, 400, 1)),
+        (np.zeros((6, 200, 400)), "ZYX", "CYSXT", {"Z": 2}, (1, 200, 1, 400, 1)),
```

Comment:
I'm really glad you expanded the tests to cover the cases extensively. As with any mathematical operation, it's better to have tests, as the code is often difficult to grok.